Dockerfile.runner

```diff
@@ -0,0 +1,66 @@
+FROM quay.io/cybozu/ubuntu-debug:20.04
+
+ARG RUNNER_VERSION=2.277.1
+ARG DUMB_INIT_VERSION=1.2.2
+ARG KUBERNETES_VERSION=1.19.7
+
+ENV DEBIAN_FRONTEND=noninteractive
+RUN apt-get update -y \
+  && apt-get install -y software-properties-common \
+  && add-apt-repository -y ppa:git-core/ppa \
+  && apt-get update -y \
+  && apt-get install -y --no-install-recommends \
+    build-essential \
+    dnsutils \
+    git \
+    iproute2 \
+    iputils-ping \
+    jq \
+    libunwind8 \
+    locales \
+    netcat \
+    openssh-client \
+    parallel \
+    rsync \
+    shellcheck \
+    sudo \
+    telnet \
+    time \
+    tzdata \
+    unzip \
+    upx \
+    wget \
+    zip \
+    zstd \
+    vim \
+    # this is need to run actions runner
+    libyaml-dev \
+  && rm -rf /var/lib/apt/lists/*
+
+RUN curl -L https://dl.k8s.io/release/v${KUBERNETES_VERSION}/bin/linux/amd64/kubectl -o /usr/local/bin/kubectl \
+  && chmod +x /usr/local/bin/kubectl
+
+RUN curl -sSLf -o /usr/local/bin/dumb-init https://github.com/Yelp/dumb-init/releases/download/v${DUMB_INIT_VERSION}/dumb-init_${DUMB_INIT_VERSION}_amd64 \
+  && chmod +x /usr/local/bin/dumb-init
+
+ARG RUNNER_ASSETS_DIR=/runner
+RUN mkdir -p ${RUNNER_ASSETS_DIR} \
+  && cd ${RUNNER_ASSETS_DIR} \
+  && curl -L -o runner.tar.gz https://github.com/actions/runner/releases/download/v${RUNNER_VERSION}/actions-runner-linux-x64-${RUNNER_VERSION}.tar.gz \
+  && tar xzf ./runner.tar.gz \
+  && rm runner.tar.gz \
+  && ./bin/installdependencies.sh \
+  && rm ./bin/RunnerService.js \
+  && chown -R 10000 ${RUNNER_ASSETS_DIR}
+
+ENV AGENT_TOOLSDIRECTORY=/opt/hostedtoolcache
+RUN mkdir -p ${AGENT_TOOLSDIRECTORY} \
+  && chmod g+rwx ${AGENT_TOOLSDIRECTORY}
+
+USER 10000
+COPY scripts/entrypoint.sh /usr/local/bin/
+COPY scripts/job-failed /usr/local/bin/
+COPY scripts/RunnerService.js ${RUNNER_ASSETS_DIR}/bin/
```

Comment:
I don't know how to run `RunnerService.js`.
Is it not necessary to install node.js?

Reply:
Yes.

As [this line](https://github.com/actions/runner/blob/8109c962f09d9acc473d92c595ff43afceddb347/src/Misc/layoutbin/runsvc.sh#L16) shows, `node` binary is placed under `/runner/externals`.

This originally comes from [the released package on actions/runner](https://github.com/cybozu-go/github-actions-controller/blob/3cd20de77f78f2108d37769eaa6f2f622591e306/Dockerfile.runner#L49).